static/js/settings.js

```diff
@@ -112,6 +112,7 @@ function setup_settings_label() {
         left_side_userlist: i18n.t("User list on left sidebar in narrow windows"),
         night_mode: i18n.t("Night mode"),
         starred_message_counts: i18n.t("Show counts for starred messages"),
+        fluid_layout_width: i18n.t("Fluid width"),
```

Comment:
@rishig can you give me some feedback about the label of this setting?

Reply:
How about "Use full width on wide screens"?

Reply:
Yeah, sounds good! 👍